lux/lib/lux/prisms/goat_sdk/uniswap_swap.ex

```diff
@@ -0,0 +1,120 @@
+# lib/lux/prisms/goat_sdk/uniswap_swap.ex
+defmodule Lux.Prisms.GoatSdk.UniswapSwap do
```

Comment:
This looks great actually and thanks for the effort of testing the python part. I'm thinking of how to make the process a little easier.

Not sure if you'd be willing to re-write this, but Lux also supports [pure python prisms](https://github.com/sojinmm/lux/blob/da30e4b5c72b54e68eead9260248bc4984be6002/lux/guides/prisms.livemd#L540-L541). We then could have simpler pure python tests, I think.

All the python action happens in the `priv` folder.